google/cloud/sql/connector/connector.py

```diff
@@ -17,6 +17,54 @@
 import googleapiclient
 
 
+def get_metadata(service, proj_name, inst_name):
+    """
```

Comment:
Sort of a higher level discussion the python team should look at. Our other modules are using sphinx style docstrings. There are two things I would want to verify if we used google style.

1) I have heard that napoleon and sphinx should be able to translate google style.
2) I would want to know that IDEs/plugins that do typehinting are also good at this. I think the sphinx style is verbose which helps out a bit. I don't use PyCharm but it seems they support sphinx style? https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html

You are welcome to leave this and if at documentation generation time we struggle, we might have to address this. just heads up.

For an example of what I mean: https://github.com/googleapis/google-cloud-python/blob/master/storage/google/cloud/storage/client.py#L35

Reply:
I'll open a PR to change the documentation to Sphinx style